pytest/tests/stress/hundred_nodes/node_rotation.py

```diff
@@ -6,9 +6,9 @@
 import os
 
 sys.path.append('lib')
-from cluster import collect_gcloud_config, GCloudNode, Key
+from cluster import GCloudNode, Key
 from transaction import sign_staking_tx
-from utils import user_name
+from utils import user_name, collect_gcloud_config
```

Comment:
if collect_gcloud_config is using in 100 nodes, I suggest to use rc.pmap, which will save some time. Just wrap this:
```
        if not os.path.exists(f'/tmp/near/node{i}'):
            # TODO: avoid hardcoding the username
            print(f'downloading node{i} config from gcloud')
            pathlib.Path(f'/tmp/near/node{i}').mkdir(parents=True, exist_ok=True)
            gcloud.get(f'pytest-node-{user_name()}-{i}').download('/home/bowen_nearprotocol_com/.near/config.json', f'/tmp/near/node{i}/')
            gcloud.get(f'pytest-node-{user_name()}-{i}').download('/home/bowen_nearprotocol_com/.near/signer0_key.json', f'/tmp/near/node{i}/')
            gcloud.get(f'pytest-node-{user_name()}-{i}').download('/home/bowen_nearprotocol_com/.near/validator_key.json', f'/tmp/near/node{i}/')
            gcloud.get(f'pytest-node-{user_name()}-{i}').download('/home/bowen_nearprotocol_com/.near/node_key.json', f'/tmp/near/node{i}/')
        with open(f'/tmp/near/node{i}/signer0_key.json') as f:
            key = json.load(f)
        keys.append(key)
```
to a func, and pmap(func, range(num_nodes).